go/libkb/util_windows.go

```diff
@@ -184,3 +188,47 @@ func RemoteSettingsRepairman(g *GlobalContext) error {
 	}
 	return nil
 }
+
+// Notify the shell that the thing located at path has changed
+func notifyShell(path string) {
+	shChangeNotifyProc.Call(
+		uintptr(0x00002000), // SHCNE_UPDATEITEM
+		uintptr(0x0005),     // SHCNF_PATHW
+		uintptr(unsafe.Pointer(syscall.StringToUTF16Ptr(path))),
+		0)
```

Comment:
Never mind, I think this is [documented to be safe](https://golang.org/pkg/unsafe/#Pointer):

> The compiler handles a Pointer converted to a uintptr in the argument list of a call to a function implemented in assembly by arranging that the referenced allocated object, if any, is retained and not moved until the call completes, even though from the types alone it would appear that the object is no longer needed during the call.